payment-method-basic-card/billing-address-is-null-manual.https.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html> <meta charset="utf-8" />
+<title>Test for requesting billing address</title>
+<link
+  rel="help"
+  href="https://github.com/w3c/payment-method-basic-card/pull/65"
+/>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  setup({
+    explicit_done: true,
+    explicit_timeout: true,
+  });
+  const basicCard = { supportedMethods: "basic-card" };
+  const details = {
+    total: {
+      label: "label",
+      amount: { currency: "USD", value: "5.00" },
+    },
+  };
+
+  // Smoke tests
+  test(() => {
+    assert_true(
+      "onpaymentmethodchange" in PaymentRequest.prototype,
+      "The paymentmethodchange event handler is not supported on PaymentRequest"
+    );
+    assert_true(
+      "PaymentMethodChangeEvent" in window,
+      "The PaymentMethodChangeEvent is not supported"
+    );
+  }, "PaymentMethodChangeEvent support");
+
+  function dontRequestBillingAddress(options) {
+    promise_test(async t => {
+      const request = new PaymentRequest([basicCard], details, {
+        requestBillingAddress: false,
```

Comment:
Is it a bug that the optional `PaymentOptions` parameter is not listed in the constructor arguments in the spec? https://w3c.github.io/payment-request/#paymentrequest-interface

Reply:
oh, sometimes it might have gotten text-wrapped onto the next line, but it's definitely there:

[Constructor(sequence<PaymentMethodData> methodData, PaymentDetailsInit details, optional PaymentOptions options),
SecureContext, Exposed=Window]